lib/sensors/mux.hpp

```diff
@@ -18,33 +18,33 @@ static constexpr core::Float kFailureThreshold
 template<class T, std::uint8_t N>
 class Mux {
  public:
-  Mux(io::II2c &i2c,
+  Mux(core::ILogger &logger,
+      io::II2c &i2c,
       const std::uint8_t mux_address,
-      std::array<std::unique_ptr<II2cMuxSensor<T>>, N> &sensors,
-      core::ILogger &log);
+      std::array<std::unique_ptr<II2cMuxSensor<T>>, N> &sensors);
   ~Mux();
 
   std::optional<std::array<T, N>> readAllChannels();
 
  private:
   core::Result selectChannel(const std::uint8_t channel);
   core::Result closeAllChannels();
```

Comment:
actually having these here looks a bit ugly - do you think these should go below the variables with a space in between? (or something else) @mifrandir

Reply:
Just put a space between class members and private methods? Wouldn't do anything else tbh.